src/workerd/api/pyodide.c++

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#include "pyodide.h"
+
+#include <capnp/common.h>
+#include <capnp/compat/json.h>
+
+namespace workerd {
```

Comment:
Since this is in `workerd/api`, let's make the namespace `workerd::api` or `workerd::api::pyodide` to align with `workerd::api::node`